mongoDB/seed.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable no-console */
```

Comment:
You could also add this to your eslint config, or are you leaving it out on purpose?